ArduPlane/parachute.cpp

```diff
@@ -56,3 +59,55 @@ bool Plane::parachute_manual_release()
 
     return true;
 }
+
+// Code to detect loss of control for ArduPlane
+#define PARACHUTE_CHECK_TRIGGER_SEC         1  // 1 second of loss of control triggers the parachute
```

Comment:
You're right. I was just replicating what was in _ArduCopter/crash_check.cpp_ (although it probably has a define in the middle of the code because of the `#if`).